DirectEmployers/MyJobs
# [PD-3081] cbo mobile ux

## What's this PR do?
Fixes the mobile version for CBO

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the CBO application

## How should this be manually tested?
Pull down the repository. Shrink your screen size and test the mobile to see that the functionality still works in mobile

## Any background context you want to provide?
Most of it was mobile responsive already just had to make a few tweaks here and there. This is about the best I could come up with for mobile but I know eventually we are re-doing the entire thing
